admin/spec/components/previews/solidus_admin/sidebar/component_preview.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "solidus_admin/main_nav_item"
+
+# @component "sidebar"
+class SolidusAdmin::Sidebar::ComponentPreview < ViewComponent::Preview
+  include SolidusAdmin::Preview
+
+  class ItemComponent < SolidusAdmin::Sidebar::Item::Component
+    def path
+      "#"
+    end
+
+    def active?
+      false
+    end
+  end
+
+  # @param store_name text
+  # @param store_url url
```

Comment:
Isn't it still `text`?

Reply:
There's an [explicit `url` version](https://lookbook.build/guide/previews/params#text) :slightly_smiling_face: